packages/svelte/src/internal/client/types.d.ts

```diff
@@ -80,6 +80,11 @@ export type SourceSignal<V = unknown> = {
 	v: V;
 };
 
+export type SourceSignalDebug = {
```

Comment:
This type feels wrong, not sure why it doesn't error - shouldn't this extend the original signal type, i.e. `{ .. } & SourceSignal`?

Reply:
I mean it worked so I just went with it